whisper/hpu_utils.py

```diff
@@ -0,0 +1,15 @@
+import warnings
+
+import torch
+
+
+def load_default_hpu() -> str:
+    """
+    Load HPU if available, otherwise use CUDA or CPU.
+    """
+
+    if not torch.hpu.is_available():
```

Comment:
Does `torch.hpu.is_available()` work (returns False) if habana module hasn't been loaded (`load_habana_module`)?

Reply:
chaned it to load the `load_habana_module` at the very begging of the file and load that file only for specific use case with running `hpu`

Reply:
Now  the container build fine. 

But when I launch it as per readme.md 

docker run -it --runtime=habana \
  -e HABANA_VISIBLE_DEVICES=all \
  -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
  --cap-add=sys_nice \
  --net=host \
  --ipc=host \
  -v /path/to/your/whisper:/workspace/whisper \
  whisper_hpu:latest \
  /bin/bash
  
  
  Then try:
  
  whisper audio.flac audio.mp3 audio.wav --model turbo --device hpu 
  
  It does not work! 
  
  ----------------
  
  Issues I notice:
  
  The Dockerfile set work dir to /root/whisper 
  
  but docker launcher map local whisper to /workspace/whisper 
  
  Is this okay?

Reply:
@orionsBeltWest Updated the README.md and Dockerfile.hpu